assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSelectionPanel/AudienceItem.js

```diff
@@ -150,8 +150,8 @@ export default function AudienceItem( {
 AudienceItem.propTypes = {
 	slug: PropTypes.string.isRequired,
 	title: PropTypes.string.isRequired,
-	description: PropTypes.string.isRequired,
-	subtitle: PropTypes.string.isRequired,
+	description: PropTypes.string,
```

Comment:
I should've caught this during IBR, but it looks like it should be sufficient to only change the prop type for the `subtitle` prop. We can leave `description` as it is.

```suggestion
	description: PropTypes.string.isRequired,
```